RecoLocalMuon/GEMSegment/plugins/ME0SegAlgoRU.cc

```diff
@@ -29,14 +29,16 @@ ME0SegAlgoRU::ME0SegAlgoRU(const edm::ParameterSet& ps)
 
 	allowWideSegments = ps.getParameter<bool>("allowWideSegments");
 	doCollisions = ps.getParameter<bool>("doCollisions");
+
 	stdParameters.maxChi2Additional = ps.getParameter<double>("maxChi2Additional");
 	stdParameters.maxChi2Prune      = ps.getParameter<double>("maxChi2Prune"     );
 	stdParameters.maxChi2GoodSeg    = ps.getParameter<double>("maxChi2GoodSeg"   ) ;
 	stdParameters.maxPhiSeeds       = ps.getParameter<double>("maxPhiSeeds"      ) ;
 	stdParameters.maxPhiAdditional  = ps.getParameter<double>("maxPhiAdditional" );
 	stdParameters.maxETASeeds       = ps.getParameter<double>("maxETASeeds"      );
 	stdParameters.maxTOFDiff        = ps.getParameter<double>("maxTOFDiff"      );
-	stdParameters.minNumberOfHits        = ps.getParameter<unsigned int>("minNumberOfHits"      );
+	stdParameters.requireCentralBX  = ps.getParameter<bool>("requireCentralBX"      );
+	stdParameters.minNumberOfHits   = ps.getParameter<unsigned int>("minNumberOfHits"      );
 	stdParameters.requireBeamConstr = true;

```

Comment:
@perrotta I addressed the other issues brought up in my last commit but did not add the "ParameterSetDescription." I understand that this will be a rather large change as the class uses a vector of parameter sets with different parameter for different algorithms. Is there a suggested way to handle this?